src/core/sys/posix/fcntl.d

```diff
@@ -98,6 +98,12 @@ version( CRuntime_Glibc )
     enum F_SETLK        = 6;
     enum F_SETLKW       = 7;
   }
+  else version(AArch64)
+  {
+    enum F_GETLK        = 5;
+    enum F_SETLK        = 6;
+    enum F_SETLKW       = 7;
+  }
```

Comment:
This turned up when trying to run [this Phobos test on linux/AArch64](https://github.com/dlang/phobos/blob/master/std/stdio.d#L1416), dug this fix out of the Glibc headers. It appears Glibc uses these constants on AArch64 regardless of whether `__USE_FILE_OFFSET64` is set or not, just like on `x64`.

I don't want to go digging through the rat's nest of platform-specific Glibc headers to figure out why this is, so I just added this exception. If someone wants to do that digging and suggest a better way to normalize this, I'd be glad to make that change.